packages/create-webstone-app/template/plugin/structure/package.json

```
@@ -0,0 +1,24 @@
+{
+	"name": "webstone-reference-plugin",
+	"version": "0.0.1",
+	"description": "Reference Plugin for webstone",
```

```suggestion
	"description": "Template Plugin for Webstone",
```

Or maybe even this?

`"Template Plugin for Webstone - PLEASE UPDATE THIS"`